src/commands/secrets/download.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+import path from 'path';
+import tmp from 'tmp';
+import untildify from 'untildify';
+import AccountUtils from '../../architect/account/account.utils';
+import { EnvironmentUtils } from '../../architect/environment/environment.utils';
+import Command from '../../base-command';
+import { SecretsDict } from '../../dependency-manager/secrets/type';
+
+tmp.setGracefulCleanup();
+
+export default class SecretsDownload extends Command {
+  async auth_required(): Promise<boolean> {
+    return false;
+  }
+
+  static description = 'Download secrets from an account or an environment';
+  static aliases = ['secrets', 'secrets/get'];
+
+  static flags = {
+    ...Command.flags,
+    ...AccountUtils.flags,
+    ...EnvironmentUtils.flags,
+  };
+
+  static args = [{
+    name: 'secrets_file',
+    description: 'Secrets filename to download secrets',
+    default: './secrets.yml',
```

Comment:
This should not have a default and should be required for the user to specify a path. The main reason is we do not accidentally want to override their secret file if they do not have  a backup.

Reply:
That makes sense. Removed and made it required.